config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -34,6 +34,7 @@ periodics:
       - --gcp-zone=us-west1-b
       - --ginkgo-parallel=8
       - --provider=gce
+      - --gcp-nodes=2
```

Comment:
Update the ginkgo.skip below to skip `[Feature:.+\]` as well (same as our gce-windows-master job)